packages/components/app/styles/components/dropdown.scss

```diff
@@ -156,171 +162,208 @@ $size-props: (
 }
 
 
-// TODO dropdown container, should this be styled here?
-.hds-dropdown__list {
-  align-items: flex-start;
-  background-color: white; //TODO REPLACE WITH TOKEN
+// LIST
+// UL ELEMENT
+// GOES INSIDE HDS::DISCLOSURE's :content block
+
+.hds-dropdown-list {
+  background-color: var(--token-color-surface-primary);
   border-radius: 6px;
-  // box-shadow: var(--token-elevation-high-box-shadow); //TODO what should this actually be?
-  box-shadow: 0px 0px 0px 1px rgba(101, 106, 118, 0.25), 0px 2px 3px rgba(101, 106, 118, 0.15), 0px 16px 16px -10px rgba(101, 106, 118, 0.2);
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
+  box-shadow: var(--token-surface-high-box-shadow);
+  box-sizing: border-box;  //TODO 
   list-style: none;
-  margin: 0;
-  min-height: fit-content;
+  margin: 6px 0 0;
```

Comment:
[question]: why this margin? (if it's to separate the element from the toggle, this should be applied as `top` value to the absolute definitions below)

Reply:
fixed